client/webserver/site/src/js/locales.ts

```diff
@@ -224,7 +241,24 @@ export const enUS: Locale = {
   [ID_CONNECT_WALLET_ERR_MSG]: 'Failed to connect {{ assetName }} wallet: {{ errMsg }}',
   [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'Redeemed by {{ makerAddr }}',
   [ID_REFUND_IMMINENT]: 'Will happen in the next few blocks',
-  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Refund will happen after {{ refundAfterTime }}'
+  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Refund will happen after {{ refundAfterTime }}',
+  [ID_AVAILABLE_TITLE]: 'Available',
+  [ID_LOCKED_TITLE]: 'Locked',
+  [ID_IMMATURE_TITLE]: 'Immature',
+  [ID_SWAPPING]: 'Swapping',
+  [ID_BONDED]: 'Bonded',
+  [ID_LOCKED_BAL_MSG]: 'Total funds temporarily locked to cover the costs of your bond maintenance, settling matches, and necessary fees',
+  [ID_IMMATURE_BAL_MSG]: 'Incoming funds awaiting confirmation',
+  [ID_LOCKED_SWAPPING_BAL_MSG]: 'Funds currently locked in settling matches',
+  [ID_LOCKED_BOND_BAL_MSG]: 'Funds locked in active bonds',
+  [ID_RESERVES_DEFICIT]: 'Reserves Deficit',
+  [ID_RESERVES_DEFICIT_MSG]: 'Difference between the available balance in your wallet and the amount that has been reserved for specific purposes',
```

Comment:
The apparent wallet balance shortcoming to maintain bonding level. If this persists, you may need to add funds to stay fully bonded.

Reply:
I have updated the message with your suggestion in commit https://github.com/decred/dcrdex/pull/2357/commits/27d2601a3a40358fbc1aee915bd9c83a979cd8e5. But is `Reserves Deficit` only for `Fidelity Bonds`?